docker-dev/docker-compose.yml

```diff
@@ -64,7 +64,7 @@ services:
         labels:
             - "traefik.backend=omscore"
             - "traefik.port=80"
-            - "traefik.frontend.rule=HostRegexp:{domain:[a-z]+}"
+            - "traefik.frontend.rule=HostRegexp:{domain:[a-z0-9.]+}"
```

Comment:
I am actually not sure if port notation will be needed when using a different port from port 80, My guess is not as ports are already specified elsewhere in the docker.yml file. As a sidenote: as far as OMS is concerned, we are connecting to the system using the normal port 80 (VM in VM in VM stuff)